sos/plugins/kubernetes.py

```diff
@@ -88,6 +88,13 @@ def setup(self):
                 "{} get -o json pv".format(kube_cmd)
             ])
 
+            # get a node description from ndoes (convenience)
+            nodes = self.get_command_output("%s get nodes -o name"
+                                            % self.oc_cmd)
+            for node in nodes['output'].splitlines():
```

Comment:
I agree that these sound redundant/unnecessary - but also they shouldn't imply a major burden for the end user (the node describe would add ~2k / ~0.5s per node), so , so I'm also fine to add them if people find them useful.

Maybe @TurboTurtle has comments about the Kuberentes plugin change. To begin with, you should replace invocations to `oc` with invocations of `kubectl`; also you might just replace the `get nodes` + `for ... describe` loop with a single `kubectl describe node` which would get them all.